### ISSUE TITLE
irc as an offline webapp

### ISSUE THREAD

**Issue by mafintosh** [mafintosh/todo#3]
discussed this with @feross and @watson in oslo airport. would be fun to prototype

**Comment by perguth** [comment]
Yeah :D And there is a nice UI licensed under MIT: https://github.com/erming/shout
